lingua_franca/internal.py

```diff
@@ -456,10 +458,19 @@ def _call_localized_function(func, *args, **kwargs):
             lang_param_index = func_params.index('lang')
             full_lang_code = None
 
+            # Check if we need to add timezone awareness to any datetime object
+            if config.inject_timezones:
+                for key, value in kwargs.items():
+                    if isinstance(value, datetime) and value.tzinfo is None:
+                        kwargs[key] = to_local(value)
+                for idx, value in enumerate(args):
+                    if isinstance(value, datetime) and value.tzinfo is None:
+                        args = args[:idx] + (to_local(value),) + args[idx + 1:]
```

Comment:
@krisgesling I am confused by this tuple's tupleyness

Reply:
Yeah I did a double take on it at first too. 
`('a')` provides a string; whilst
`('a',)` provides a single item tuple

I've switched it out to use iterable unpacking:
```
args = (*args[:idx], to_local(value), *args[idx + 1:])
```

hopefully this is more readable, though I also don't see it being used a lot so may still confuse some people.

Reply:
It's significantly more readable, in that I thought the outer type was a list, but now I'm wondering why `args` is a tuple 👅